src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.props

```diff
@@ -4,5 +4,6 @@
 		<_JavaInteropReferences>Java.Interop;System.Runtime</_JavaInteropReferences>
 		<DependsOnSystemRuntime Condition=" '$(DependsOnSystemRuntime)' == '' ">true</DependsOnSystemRuntime>
 		<CopyNuGetImplementations Condition=" '$(CopyNuGetImplementations)' == ''">true</CopyNuGetImplementations>
+		<_YieldDuringExecution Condition="'$(_YieldDuringExecution)' == ''">true</_YieldDuringExecution>
```

Comment:
Because that feels like polluting the "global namespace" with a symbol we don't and shouldn't own. :-)